frontend/packages/dev-console/integration-tests/features/Helm/actions-on-HelmRelease.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Helm Chart
+    User should able to perform the actions on the helm releases
+
+Background:
+    Given user logged into the openshift application
+    And user is at developer perspecitve
+
+
+@regression, @smoke
+Scenario: Perform Upgrade action on Helm Release through Context Menu: HR-08-TC02
+    Given user right clicks on the Helm Release
+    When user clicks on the Upgrade action
+    And user updates the chart Version
+    And user clicks on the upgrade button
+    Then the helm release should get upgradaed 
+    And user should get redirected to topology page
+
+
+@regression, @smoke
+Scenario: Perform Rollback action on Helm Release through Context Menu: HR-08-TC03
+    Given user right clicks on the Helm Release
+    When user clicks on the Rollback action
+    And user selects the version to Rollback
+    And user clicks on the rollback button
+    Then the helm release should rollback to the version
```

Comment:
No Assertion words in the feature files please

Reply:
Updated accordingly.